CMakeLists.txt

```diff
@@ -677,6 +694,54 @@ elseif(APPLE)
         ${CMAKE_COMMAND} -E remove_directory
         ${SDK_INSTALL_DIR}
     )
+
+# Need to make Mac-specific changes to the dylib install_name for
```

Comment:
I noticed these entries last week. Unless I'm doing something wrong, it doesn't work.

I added "dll-path=@executable_path/../SharedSupport" to the "APPEND BOOST_CONFIGURE_FLAGS" (at line 530).
The install_names of the boost shared libraries did not change. They are still in the format of @rpath/libboost_(name).dylib, with nothing to indicate that I set "dll_path" to something different.